src/qt/optionsdialog.cpp

```diff
@@ -295,10 +295,22 @@ void OptionsDialog::on_resetButton_clicked()
 
 void OptionsDialog::on_openBitcoinConfButton_clicked()
 {
-    /* explain the purpose of the config file */
-    QMessageBox::information(this, tr("Configuration options"),
-        tr("The configuration file is used to specify advanced user options which override GUI settings. "
-           "Additionally, any command-line options will override this configuration file."));
+    QMessageBox config_msgbox;
```

Comment:
To preserve behavior
```suggestion
    QMessageBox config_msgbox(this);
```

https://doc.qt.io/qt-5/qdialog.html:
> A dialog is always a top-level widget, but if it has a parent, its default location is centered on top of the parent's top-level widget (if it is not top-level itself).

Reply:
Addressed in  0b869df